challenge9.js

```diff
@@ -0,0 +1,12 @@
+const removeDuplicate = ( string ) => {
+    let newString = ``,index=0,letter;
+    for ( index; index < string.length; index++ ) {
+        if ( !newString.includes(string[index]) ) {
+            newString += string[index];
+        }
+    }
```

Comment:
Maybe just use `Set` like this
```suggestion
    const newString = Array.from(new Set(string)).join('');
```